### DISCUSSION TITLE
Admin UI: Extendable List of Admin Settings

### DISCUSSION THREAD

**Discussion by devianadim** [getdokan/storegrowth-sales-booster#428] [Bug / Problem]
### After Release Candidates:

- [ ] Implement quick cart popups font color settings option. [Link](https://team-invizo.slack.com/archives/C051BCFJV6Z/p1699851514580359)
- [ ] Implement sales popup store name font color settings option.
- [ ] Implement sales countdown font color settings option.

**Comment by imtiaz-pranto** [comment]
Converting it into discussion for further processing
